translations/stripes-smart-components/en.json

```diff
@@ -91,6 +91,7 @@
   "searchReturnedResults": "Search returned {count, number} {count, plural, one {result} other {results}}",
   "searchResultsCountHeader": "{count, number} {count, plural, one {record found} other {records found}}",
   "searchResultsCountUnknown": "More than 10,000 records found",
+  "searchResultsSelectedCount": "{count, number} {count, plural, one {record selected} other {records selected}}",
```

Comment:
I think `SearchAndSort` should not bother about specifics of what should be displayed in the subheader. For `ui-inventory` it is selected record count but who knows, what it might be for the other place. So I suggest adding some basic prop to be able to pass there some react node. And that is it. The [delimiter](https://github.com/folio-org/stripes-smart-components/pull/969/files#diff-9321b7b5e5671b9de9cae523d45f8aaced8f9a203162ddcfc90bfb4f4aa9db33R17-R20) should be in place though.